framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1291,6 +1293,17 @@ public Map<VariableElement, Value> getFinalLocalValues() {
         return flowResult.getFinalLocalValues();
     }
 
+    /**
+     * Returns true if the receiver of a method or constructor might not yet be fully initialized.
```

Comment:
This takes the declaration of a method/constructor. What does the "not yet" refer to here? Either that declaration has annotations on the receiver parameter or not.